### DISCUSSION TITLE
How to prevent teleport agent from auto-generating Azure tags?

### DISCUSSION THREAD

**Discussion by kangeek** [gravitational/teleport#24030] [Q&A] [answered]
We connect Azure VM and AKS with teleport-ssh-agent and teleport-kube-agent, and it works well.

However, we do not want to display all of the auto generated Azure tags, shown as blew:

<img width="1298" alt="image" src="https://user-images.githubusercontent.com/7991571/229690064-d5835655-a2ab-4cfd-96f9-f84fac3ac09c.png">

Sometimes it can be difficult to see the needed tags at a glance, and neither neat nor clear:

<img width="1273" alt="image" src="https://user-images.githubusercontent.com/7991571/229690626-6f004e37-db5b-459b-85fb-b0b164d26fba.png">

So, is it possible to prevent teleport agent from auto-generating Azure tags? or can we specify which tags to be hidden or displayed?

Thanks, and have a nice day!

**Comment by webvictim** [comment, marked as answer]
Unfortunately there's no way to limit which AWS/Azure/GCP tags are automatically shown at present. I raised https://github.com/gravitational/teleport/issues/24230 to track this request.

**Reply by kangeek** [reply]
Thanks!
